### PR TITLE
bugfix: S3C-1922_Remove_scality/s3server_refs

### PR DESCRIPTION
**Pull request by wabernat**
S3Server references were out of date (changed to CloudServer).
Fixing them revealed most of the links were broken, a lot of the
language was inelegant, and the organization made the document
hard to follow. Hence the voluminous corrections.

Signed-off-by: William Abernathy <william.abernathy@scality.com>

### CODE REVIEW

**Comment by wabernat** [comment]
@smaffulli: Please review this. If you cannot get to it, please let me know, and we'll move ahead without your input.

File: docs/GETTING_STARTED.rst [hunk 1]
```diff
@@ -6,210 +6,206 @@ Getting Started
...
-    npm start
+   $ mkdir -m 700 $(pwd)/myFavoriteDataPath
+   $ mkdir -m 700 $(pwd)/myFavoriteMetadataPath
+   $ export S3DATAPATH="$(pwd)/myFavoriteDataPath"
+   $ export S3METADATAPATH="$(pwd)/myFavoriteMetadataPath"
+   $ npm start
 
-Run it with multiple data backends
-----------------------------------
+Running Cloudserver with Multiple Data Backends
```
Comment (smaffulli) [inline docs/GETTING_STARTED.rst]: typo: CloudServer, with capital S

File: docs/GETTING_STARTED.rst [hunk 2]
```diff
@@ -6,210 +6,206 @@ Getting Started
...
+constraint determines where the data is saved. If the bucket has no
+location constraint, the endpoint of the PUT request determines location.
 
-See the Configuration section below to learn how to set location
-constraints.
+See the Configuration_ section to set location constraints.
 
-Run it with an in-memory backend
---------------------------------
+Run Cloudserver with an In-Memory Backend
```
Comment (smaffulli) [inline docs/GETTING_STARTED.rst]: CloudServer, capital S

File: docs/GETTING_STARTED.rst [hunk 3]
```diff
@@ -6,210 +6,206 @@ Getting Started
...
-This starts an Zenko CloudServer on port 8000. The default access key is
-accessKey1 with a secret key of verySecretKey1.
+This starts a Zenko CloudServer on port 8000. 
 
-Run it for continuous integration testing or in production with Docker
-----------------------------------------------------------------------
+The default access key is accessKey1. The secret key is verySecretKey1.
 
-`DOCKER <../DOCKER/>`__
+Run Cloudserver for Continuous Integration Testing or in Production with Docker
```
Comment (smaffulli) [inline docs/GETTING_STARTED.rst]: CloudServer :)

File: docs/INTEGRATIONS.rst [hunk 4]
```diff
@@ -4,510 +4,436 @@ Integrations
...
 
-`Docker swarm <https://docs.docker.com/engine/swarm/>`__ is a
-clustering tool developped by Docker and ready to use with its
-containers. It allows to start a service, which we define and use as a
-means to ensure Zenko CloudServer's continuous availability to the end user.
-Indeed, a swarm defines a manager and n workers among n+1 servers. We
-will do a basic setup in this tutorial, with just 3 servers, which
-already provides a strong service resiliency, whilst remaining easy to
-do as an individual. We will use NFS through docker to share data and
+`Docker Swarm <https://docs.docker.com/engine/swarm/>`__ is a clustering tool
```
Comment (smaffulli) [inline docs/INTEGRATIONS.rst:7]: Do we still test and support Docker Swarm?

**Review by smaffulli — DISMISSED** [review]
Some minor fixes

File: docs/INTEGRATIONS.rst (hunk 4 above)
Comment (wabernat) [reply docs/INTEGRATIONS.rst:7]: I'm going to ignore this for now, only because it exceeds the scope of the ticket. I believe you are 100% correct--we use Kubernetes do do this job now, so these instructions are in all likelihood wrong, or at least irrelevant. Unfortunately, I can't fix it by myself. 

Please open a ticket to this effect.

File: docs/GETTING_STARTED.rst (hunk 1 above)
Comment (wabernat) [reply docs/GETTING_STARTED.rst]: fixed

File: docs/GETTING_STARTED.rst (hunk 3 above)
Comment (wabernat) [reply docs/GETTING_STARTED.rst]: fixed

File: docs/GETTING_STARTED.rst (hunk 2 above)
Comment (wabernat) [reply docs/GETTING_STARTED.rst]: fixed

**Review by rahulreddy — APPROVED** [review]

**Review by JianqinWang — APPROVED** [review]

**Review by ssalaues — APPROVED** [review]

**Review by anurag4DSB — APPROVED** [review]

**Comment by wabernat** [comment]
@bert-e approve
